controller/src/main/java/io/pravega/controller/store/PravegaTablesStoreHelper.java

```diff
@@ -244,6 +249,89 @@ public void invalidateCache(String table, String key) {
                 .whenComplete((r, ex) -> releaseEntries(entries));
     }
 
+    public CompletableFuture<List<Version>> removeValues(final String tableName, Set<String> tableKeys, final String appendValue, long requestId) {
```

Comment:
please add java doc explaining what this method is about. 
i believe its taking some keys in a table segment and doing a get and put on the value against those keys, right?
when this is a removeValues method, why does it have an append value as a param?

Reply:
fixed it.